apps/chef_objects/src/chef_key.erl

```diff
@@ -0,0 +1,138 @@
+%% -*- erlang-indent-level: 4;indent-tabs-mode: nil -*-
+%% ex: ts=4 sw=4 et
+%% @author Tyler Cloke <tyler@chef.io>
+%% Copyright 2015 Chef Software, Inc. All Rights Reserved.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+
+-module(chef_key).
+
+-behaviour(chef_object).
+
+-export([
+         authz_id/1,
+         is_indexed/0,
+         ejson_for_indexing/2,
+         update_from_ejson/2,
+         set_created/2,
+         fields_for_update/1,
+         fields_for_fetch/1,
+         ejson_from_list/2,
+         record_fields/0,
+         list/2,
+         set_updated/2,
+         new_record/3,
+         name/1,
+         id/1,
+         org_id/1,
+         type_name/1
+        ]).
+
+%% database named queries
+-export([
+         create_query/0,
+         update_query/0,
+         delete_query/0,
+         find_query/0,
+         bulk_get_query/0,
+         list_query/0
+        ]).
+
+-include_lib("mixer/include/mixer.hrl").
+-mixin([{chef_object,[
+                     {default_fetch/2, fetch},
+                     {default_update/2, update}
+                    ]}]).
+
+-ifdef(TEST).
+-compile(export_all).
+-endif.
+
+-include("../../include/chef_types.hrl").
+-include_lib("eunit/include/eunit.hrl").
```

Comment:
We seem to be a bit inconsistent about whether we want to include_lib in the source files or not.  Do we need this here?